Engine/ComponentSettingsContext.cs

```diff
@@ -164,8 +164,7 @@ public ComponentSettings GetCurrent(Type settingsType)
             return objectCache.Invoke(settingsType);
         }
 
-
-        public void SetCurrent(Stream xmlFileStream)
+        public XmlError[] SetCurrent(Stream xmlFileStream)
```

Comment:
This is not considered breaking, because of the wrapper?

Reply:
Yes, `ComponentSettingsContext` is an internal class